File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/PropertiesBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/15
 */
@NoArgsConstructor
@Data
public class PropertiesBean {
    private String driver_class_name;
    private String url;
    private String password;
    private String username;
    private String cipherType;
    private String cipherKey;

    public String unique() {
        return url + ":" + username + ":" + password + ":" + driver_class_name;
    }
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/PropertiesBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/15
 */


@AllOpen
class PropertiesBean {
    var driver_class_name: String? = null
    var url: String? = null
    var password: String? = null
    var username: String? = null
    var cipherType: String? = null
    var cipherKey: String? = null

    fun unique(): String {
        return "$url:$username:$password:$driver_class_name"
    }
}
